diagnostic.md

```diff
@@ -16,13 +16,13 @@ Graces-MacBook-Pro:try grace$
 
 What is the full path to the current directory?
 
-_Replace this text with your answer._
+/Users/grace/wdi/try
 
 ## Question 2
 
 Describe GitHub, **briefly** in your own words.
 
-_Replace this text with your answer._
+It is a collection of git repositories located in the cloud.
```

Comment:
Yes, but more specifically it is 'A user interface (UI) for accessing git's version control capabilities for our projcet repositories'.